### DISCUSSION TITLE
French translation of the UI

### DISCUSSION THREAD

**Discussion by arcqus** [TriliumNext/Trilium#4726] [Q&A] [answered]
Hello
After a few hours of use, I think trilium is the best note taking app that I used so far.
I don't see a french translation of the UI. Am I missing something or is it something to do ?
If missing, here is a few questions :
1) does it requires any technical knowledge ?
2) what are the required tools ? 
3) what is the amount of work (ie number of text bits to translate)
I have a limited amount of available of time but would be glad to help if it is possible with limited technical knowledge.

**Comment by Nriver** [comment]
The official Trilium does not provide translation. But I've made an unofficial project to translate it to other language https://github.com/Nriver/trilium-translation.

1. Yes, it does.
2. You can read the `how to translate` part of the readme file.
3. You can refer to this file https://github.com/Nriver/trilium-translation/blob/main/translations.py . It would be around 1600 terms you need to translate.

**Reply by arcqus** [reply]
Hi Nriver
Thanks for the answer.
Actually, the purpose of my suggestion is to get, in the end, the official trilium app translate, not to have a custom version.

**Reply by Nriver** [reply, marked as answer]
This has been discussed many times, and there will not be an official release for translation from this repository. Furthermore, Trilium has entered maintenance mode (https://github.com/zadam/trilium/issues/4620).

The good news is that the community has formed a new organization here: https://github.com/orgs/TriliumNext/discussions. You may find luck there.

**Reply by arcqus** [reply]
Thanks for the summary of the recent events !
A bit sad for what I read there, I'll give a look to TriliumNext.
